src/plays/schulte-tables/SchulteTables.jsx

```diff
@@ -0,0 +1,23 @@
+import './styles.css';
+import PlayHeader from 'common/playlists/PlayHeader';
+import GameTable from './GameTable';
+
+// WARNING: Do not change the entry componenet name
+function SchulteTables(props) {
+  // Your Code Start below.
+
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
+          {/* Your Code Starts Here */}
```

Comment:
Could you tell us about the comment?

Reply:
I didn't remove the initial comment mistakenly.